modules/cli-options/src/main/scala/scala/cli/commands/RunOptions.scala

```diff
@@ -29,7 +29,11 @@ final case class RunOptions(
     command: Boolean = false,
   @Group("Run")
   @HelpMessage("Temporary / working directory where to write generated launchers")
-    scratchDir: Option[String] = None
+    scratchDir: Option[String] = None,
+  @Group("Run")
+  @Hidden
```

Comment:
eh, same thing on making it hidden as with `PackageOptions.mainClassInManifest`
maybe we could mark these obscure, border-arcane options for niche usecases somehow?
so we remember to document them later in a guide/cookbook/some specialised help command.

treat this comment as thinking aloud rather than a suggestion for a change in this PR

Reply:
I left it hidden just like `PackageOptions.mainClassInManifest`… This can be changed / perfected later.